### PR TITLE
Logout user from OP.

### PR DESCRIPTION
**Pull request by akatsoulas**
* Support post http method on logout.

### CODE REVIEW

**Comment by akatsoulas** [comment]
@johngian r?

File: mozilla_django_oidc/contrib/auth0/utils.py [hunk 1]
```diff
@@ -1,4 +1,7 @@
 import requests
+import urllib
```
Comment (johngian) [inline mozilla_django_oidc/contrib/auth0/utils.py]: Please make this import python 3 friendly

```
try:
    from urllib import urlencode
except ImportError:
    from urllib.parse import urlencode
```

File: mozilla_django_oidc/views.py [hunk 2]
```diff
@@ -104,14 +105,21 @@ def get(self, request):
...
+    def dispatch(self, request, *args, **kwargs):
+        """Log out the user."""
+
+        if request.user.is_authenticated():
+            auth.logout(request)
+
+            logout_view_path = import_from_settings('OIDC_OP_LOGOUT_VIEW', '')
+            if logout_view_path:
+                logout_view = import_string(logout_view_path)
+                logout_view(request)
```
Comment (johngian) [inline mozilla_django_oidc/views.py]: This one should be ``return logout_view(request)``.

File: mozilla_django_oidc/contrib/auth0/utils.py [hunk 3]
```diff
@@ -18,3 +24,13 @@ def refresh_id_token(id_token):
     if response.status_code == requests.codes.ok:
         return response.json().get('id_token')
     return None
+
+
+def logout(request):
+    """Log out the user from Auth0."""
+    url = 'https//' + import_from_settings('OIDC_OP_DOMAIN') + '/v2/logout'
+    url += '?' + urllib.urlencode({
```
Comment (johngian) [inline mozilla_django_oidc/contrib/auth0/utils.py]: Please use urlencode.

**Pull request merged**